experiments/get_recipes_from_minecraft_data.py

```diff
@@ -0,0 +1,50 @@
+import minecraft_data
```

Comment:
Why do we have `examples/get_recipes_from_minecraft_data.py` both in `examples` and `experiments`?

Reply:
I still see two of them in this PR.